scipy/optimize/_tstutils.py

```diff
@@ -704,17 +704,17 @@ def fun4(x):
 
 
 def fun5(x):
-    return x**9
+    return x**9.
```

Comment:
These are no longer needed because `_chandrupatla` ensures that the argument is always a float.
```suggestion
    return x**9
```